components/math/MultiplicationBlock.tsx

```diff
@@ -4,22 +4,26 @@ export interface MultiplicationBlockProps {
   text: string;
   selected: boolean;
   onClick: boolean;
+  player: boolean;
 }
 
 export const MultiplicationBlock: React.FC<MultiplicationBlockProps> = ({
   text,
   onClick,
+  player,
```

Comment:
You defined this new prop but you're not using it anywhere. If you read this prop inside your ternary statement you wouldn't need to create a duplicate blocks component.